WalletWasabi/Exceptions/UnknownRoundEndingException.cs

```diff
@@ -6,15 +6,17 @@ namespace WalletWasabi.Exceptions;
 
 public class UnknownRoundEndingException : Exception
 {
-	public UnknownRoundEndingException(ImmutableList<SmartCoin> coins, ImmutableList<Script> outputScripts, Exception exception) :
+	public UnknownRoundEndingException(ImmutableList<SmartCoin> coins, ImmutableList<Script> outputScripts, Transaction? unsignedCoinJoin, Exception exception) :
```

Comment:
You are throwing this exception after signing phase, what means the transaction can never be null.

Reply:
Visual Studio disagrees with you. 

https://github.com/molnard/WalletWasabi/blob/processcj/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L259